CHANGELOG.md

```diff
@@ -1,5 +1,344 @@
+
+    [xtuc]: https://github.com/xtuc
+    [pull/193]: https://github.com/cloudflare/wrangler/pull/193
+
+- ### 🤕 Fixes
+
+  - **Use rust template for rust project - [EverlastingBugstopper], [pull/309]**
+
+    Previously, when passing `--type rust` to `wrangler generate`, the only indication that it worked was that the type in `wrangler.toml` was `rust`. There were no Rust files in the default template, for a Rust-type project. Now, when running `wrangler generate --type rust`, `wrangler` will use [rustwasm-worker-template](https://github.com/cloudflare/rustwasm-worker-template) when generating a new project.
+
+  [pull/309]: https://github.com/cloudflare/wrangler/pull/309
+
+  - **Stop cleaning webpack build artifacts - [EverlastingBugstopper], [pull/307]**
+
+    The configuration for Webpack projects in Wrangler was over-eager to clean build artifacts, which in the worst-case, caused Wrangler to remove source code for developers' projects - oops! This fix relaxes the Webpack cleaning process, to ensure that building a Wrangler project is a bit safer.
+
+  [pull/307]: https://github.com/cloudflare/wrangler/pull/307
+
+  - **Correct binding format - [xtuc], [pull/260]**
+
+    Previously, `wrangler` was incorrectly sending up a `binding` object[] to the Cloudflare API, whenever we attempted to update a script's bindings. This fix renames it to `bindings`, and uses an array, as per the Cloudflare API requirements.
+
+    [xtuc]: https://github.com/xtuc
+    [pull/260]: https://github.com/cloudflare/wrangler/pull/260
+
+  - **Correctly pass Wasm module - [xtuc], [pull/261]**
+
+    To ensure that a wasm module is successfully passed between `wranglerjs` and `wrangler`, the wasm module is now encoded and decoded from base64, avoiding any potential loss of data.
+
+    [xtuc]: https://github.com/xtuc
+    [pull/261]: https://github.com/cloudflare/wrangler/pull/261
+
+  - **Check for `account_id` and `zone_id` before publishing - [xtuc], [issue/170][pull/192]**
+
+    The `publish` subcommand in `wrangler` now ensures that you have an `account_id` and `zone_id` configured in your `wrangler.toml` file before attempting to publish, instead of failing during the publishing process.
+
+    [xtuc]: https://github.com/xtuc
+    [issue/170]: https://github.com/cloudflare/wrangler/issues/170
+    [pull/192]: https://github.com/cloudflare/wrangler/pull/192
+
+  - **Pass wranglerjs output as ref - [xtuc], [pull/227]**
+
+    When `wranglerjs` built a project, it incorrectly referred to the output of that build process without using a Rust reference - this PR fixes that issue and allows `wranglerjs` to correctly take your bundle, and your project's metadata, and put it all together in a nice little package to send up to the Cloudflare API. Hooray, working projects!
+
+    [xtuc]: https://github.com/xtuc
+    [pull/227]: https://github.com/cloudflare/wrangler/pull/227
+
+  - **Fix installer - [xtuc], [pull/195]**
```

Comment:
this is for the npm installer not wranglerjs, and I think it went out as the 1.0.2 update to NPM 2 months ago so we could probably just leave it off as the installer and wrangler are separate

Reply:
ok, works for me!